tests/tests_geomstats/test_discrete_curves.py

```diff
@@ -350,6 +337,21 @@ def test_srv_norm(self):
         expected = [srvs_ab.shape[0]]
         self.assertAllClose(result, expected)
 
+    def test_f_transform(self):
```

Comment:
🎉 fantastic! could we add a test checking that distances between two curves coincide too?